test/system/liquidation.test.js

```diff
@@ -136,6 +138,22 @@ describeFn("System -- liquidation", () => {
 
     before(async () => {
       await tbtcDeposit.notifyRedemptionSignatureTimedOut()
+
+      // The deposit's auction must offer at least 75% of bonds to be accepted
+      // by the risk manager. At starting block, the deposit's auction exposes
+      // 66% so an immediate `notifyLiquidation` must revert.
+      await expect(
+        riskManagerV1.notifyLiquidation(tbtcDeposit.address)
+      ).to.revertedWith(
+        "Deposit bond auction percentage is below the threshold level"
+      )
```

Comment:
This could be a separate test that the `notifyLiquidation` call is reverted. WDYT?

Reply:
It's not that easy as system tests should ideally use `before` not `beforeEach` which makes test initialization very hard and cumbersome.

Reply:
I'll leave this thread open as a reminder, just in case we discover a better way one day.